src/features/trainers/angular.data.ts

```diff
@@ -1,12 +1,11 @@
 import type { Trainer } from './trainers.types';
-import angImg7 from '@/assets/mentors/a-sacca.webp';
+import angImg6 from '@/assets/mentors/a-sacca.webp';
 import angImg1 from '@/assets/mentors/a-serhiyenia.webp';
-import angImg5 from '@/assets/mentors/d-kohut.webp';
-import angImg4 from '@/assets/mentors/d-yarmoshkin.webp';
+import angImg4 from '@/assets/mentors/d-kohut.webp';
```

Comment:
I'm not sure that we have to rename files with photo. If it were me, I would name the photos not by numbers, but by the trainer's name in camelCase.

Reply:
Done